modules/client/examples/00-client/02-create-dao.ts

```diff
@@ -9,20 +9,23 @@ import {
   GasFeeEstimation,
   ICreateParams,
 } from "@aragon/sdk-client";
+import { IMetadata } from "../../src";
 import { contextParams } from "./00-context";
 
 const context: Context = new Context(contextParams);
 const client: Client = new Client(context);
+const metadata: IMetadata = {
+  name: "My DAO",
+  description: "This is a description",
+  avatar: "",
+  links: [{
+    name: "Web site",
+    url: "https://...",
+  }],
+};
+const ipfsHash = await client.methods.pinMetadata(metadata);
 const createParams: ICreateParams = {
-  metadata: {
-    name: "My DAO",
-    description: "This is a description",
-    avatar: "",
-    links: [{
-      name: "Web site",
-      url: "https://...",
-    }],
-  },
+  metadataUri: ipfsHash,
```

Comment:
It does not matter, I was planning on supporting both options, if it commes with `ipfs://` we are good, if it comes without it and is a valid cid, we add it, however this is easier to do with `yup` because I can add a validation an transformation to a custom schema called `ipfsCidOrUri`

Reply:
well, the result of  `pinMetadata` should be a URI IMO

Reply:
ok then I'll remove all the `ipfs://`